components/landing/TheQuickStart/StartOnTheCloud.vue

```diff
@@ -30,8 +30,6 @@ export default class StartOnTheCloud extends Vue {
 <style lang="scss" scoped>
 .start-on-the-cloud {
   &__wrapper {
-    @include responsive-grid-bg('/images/grid/grid-white.svg', 36rem);
-
```

Comment:
Nice catch, @techtolentino! I didn't see the change in the design at first.

We need to confirm with @JRussellHuffman if this is intended. But if it is, please also:

1. delete the image in **static/images/grid/grid-white.svg**, as it's no longer used, and
2. remove the `background-position` (also from the `@include` scope in line 41) and `background-repeat` attributes, as they have no longer an effect.

Reply:
following our conversation in Slack, this will stay for now until